revisions.html.md.erb

```diff
@@ -3,7 +3,7 @@ title: Cloud Foundry API app revisions
 owner: CAPI
 ---
 
-A revision represents code and configuration used by an app at a specific time. It is a Cloud Foundry API (CAPI) object that can contain references to a droplet, a custom start command, and environment variables. The most recent revision for a running app represents code and configuration running in <%= vars.platform_name %>.
+A revision represents code and configuration used by an app at a specific time. It is a Cloud Foundry API (CAPI) object that can contain references to a droplet, a custom start command, and environment variables. The most recent revision for a running app represents code and configuration running in <%= vars.platform_name %>. Revisions are not created for Tasks.
```

Comment:
Hey @Benjamintf1 -- to spell this out a bit more, would it be accurate to say:
> Revisions of an app do not include any tasks run on the app.

Thanks!

Reply:
I was a little confused by the wording. But yeah, tasks run on the app will not create revisions.